### ISSUE TITLE
Update automatically the spec

### ISSUE THREAD

**Issue by dannywillems** [galenframework/galen#433]
Is there a way to automatically update the spec with a reference page?

Suppose you test A on page P with the spec S and everything is OK. Now, you decide to change the design of your page P and you want to run a test B. Is there a way to automatically update the spec S for the test B, without rewriting the spec S?
For example, I think about a way to change the spec S after validated a screenshot with the test B.

**Comment by ishubin** [comment]
@dannywillems I don't think it will be that easy to implement. But at the moment I am working on another project which is a storage of image samples and it will help to easily update images after a test run. But this is only for image comparison
